src/input/input.jsx

```diff
@@ -30,22 +33,31 @@ const Input = ({
   return (
     <div className={classes} style={style}>
       {label && <label htmlFor={name}>{label}{required && '*'}</label>}
-      <Field
-        className={styles.field}
-        name={name}
-        component={getComponent(type)}
-        type={type}
-        placeholder={placeholder}
-        disabled={disabled}
-        required={required}
-      >
-        {type === 'select' ?
-          options.map(o => <option key={o.id} value={o.id}>{o.label}</option>)
-          : null
-        }
-      </Field>
-      {type === 'select' &&
-        <i className={classnames(styles.arrow, 'mdv-expand_more')} />
+      {
+        type === 'selectbox' ?
+          <Select
```

Comment:
Move subelement creation selection before the return.
To avoid making this into JSX, for readability.

Common props should be in a `commonProps ` object that will be spread select and field.